mne/time_frequency/tests/test_multitaper.py

```diff
@@ -3,7 +3,7 @@
 from numpy.testing import assert_array_almost_equal
 from distutils.version import LooseVersion
 
-from mne.time_frequency import dpss_windows, multitaper_psd
+from mne.time_frequency.multitaper import dpss_windows, _psd_multitaper
```

Comment:
yeah that's all I use it for...just quick "does this function work at all" kinds of stuff